bin/nrnivmodl.in

```diff
@@ -94,21 +94,22 @@ fi
 cd $MODSUBDIR
 mdir=`pwd`
 for i in $files; do
-    rm -f "`basename $i`".mod
-    ln -s ../$i.mod .
+	rm -f "`basename $i`".mod
+	ln -s ../$i.mod .
 done
+
 if test "$incs" != "" ; then
 	for i in $incs ; do
-	    rm -f "`basename $i`"
-	    ln -s ../$i .
+		rm -f "`basename $i`"
+		ln -s ../$i .
 	done
 fi
 
 bfiles=`echo "$mfiles" | sed 's/\.mod//g'`
 
 for i in $cfiles; do
```

Comment:
if the folder where you type nrnivmodl has some c files that you want compiled and linked,
then you can list them in the cfiles environment variable.